languages/python/sqlalchemy-oso/sqlalchemy_oso/signal.py

```diff
@@ -0,0 +1,118 @@
+"""SQLAlchemy signal for processing queries."""
```

Comment:
Just confirming, has this been moved unmodified?

Reply:
That is correct